bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -1398,6 +1458,6 @@ public JsonEqualizer getEqualizer(Device device) throws IOException, URISyntaxEx
     public void SetEqualizer(Device device, JsonEqualizer settings) throws IOException, URISyntaxException {
         String postData = gson.toJson(settings);
         makeRequest("POST", alexaServer + "/api/equalizer/" + device.serialNumber + "/" + device.deviceType, postData,
-                true, true, null);
+                true, true, null, false);
     }
-}
+}
```

Comment:
Please keep a newline at the end of files? What happened it seems you are reintroducing some incorrect details?

Reply:
Done